src/main/java/com/minecolonies/entity/ai/citizen/farmer/FieldView.java

```diff
@@ -0,0 +1,129 @@
+package com.minecolonies.entity.ai.citizen.farmer;
+
+import com.minecolonies.util.BlockPosUtil;
+import com.minecolonies.util.BlockUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.util.BlockUtils'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)